pepper-apis/src/main/java/org/broadinstitute/ddp/model/familyhistory/familyHistoryMapping.conf

```diff
@@ -0,0 +1,96 @@
+// Sample section of study builder configuration that would provid configuration to establish
+// mapping between activity instance data and serialized family history model
+{
+  "familyHistory":
+  // Need to point out where participant gets its medical history
+    {
+      "participantMedicalHistory": {
+        "type": "ACTIVITY",
+        "activityCode": "MEDICALHISTORY"
+      },
+
+      // Father and mother have their dedicated activities in current Family History mockups
+      "relationMappings": [
+        {
+          // Relation in this case is "hard-coded
+          // Note that these should match the FamilyHistoryRelation enums
+          "relation": "FATHER",
+          "medicalHistory": {
+            "type": "ACTIVITY",
+            "activityCode": "FATHERMEDHISTORY"
+          }
+        },
+        //  Same as above
+        {
+          "relation": "MOTHER",
+          "medicalHistory": {
+            "type": "ACTIVITY",
+            "activityCode": "MOTHERMEDHISTORY"
+          }
+        },
+        // The family relation now comes from an answer to a question.
+        // In mockups, the relations for siblings come from pull-downs.
+        // Assuming here that the pull-down values match the enum names
+        // Note that each instance of the activity will result in a new member added to family
+        {
+          "relationDiscriminator": {
+            "stableId": "SIBLINGTYPE",
+            "medicalHistory": {
+              "type": "ACTIVITY",
+              "activityCode": "SIBLINGMEDHISTORY"
+            }
+          },
+        },
+        // This one is a little less clear at this point how to determine sex
+        // It is possible we might have to add a question mapping for sex
+        {
+          "relationDiscriminator": {
+            "stableId": "CHILD",
+            "medicalHistory": {
+              "type": "ACTIVITY",
+              "activityCode": "CHILDMEDHISTORY"
+            }
+          }
+        },
+        {
+          "relationDiscriminator": {
+            "stableId": "AUNTUNCLETYPE",
+            "medicalHistory": {
+              "type": "ACTIVITY",
+              "activityCode": "AUNTUNCLEMEDHISTORY"
```

Comment:
It could be possible for all relationships to share `activityCode`, right?  Or if we wanted to ask different questions of different relationships, we could do that too.

Reply:
Yes! We could potentially have just one activity that could be used to enter the medical history for every relation. The every activity **instance** would provide the medical history for the relation that was entered.
The mappings I used here are specifically trying to show how we would map Erin's family history mockups. There is a different survey for mother,  father, siblings, etc. Siblings of all types (full sister,paternal half-brother,etc) share the same survey.